reference/auth.v1.yaml

```diff
@@ -680,6 +680,144 @@ paths:
         - sessionToken: []
       tags:
         - Internal
+  /v1/consents:
+    get:
+      summary: List Consents
+      tags:
+        - Internal
+        - Consent
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: ./auth/models/consent/consents.v1.yaml
+      operationId: ListConsents
+      description: 'Retrieves the list of all different consent types and their content. '
+      requestBody:
+        content: {}
+      parameters: []
```

Comment:
looks like `type` is indeed a parameter.

also `type` and `latest` together  can be used but not `latest` by itself

also `page` (integer)

also `size` (integer)

Reply:
and `type` and `version`